demo/server/src/auth/local.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express';
+
+export function loginRequired(req: express.Request, res: express.Response, next: Function) {
```

Comment:
it will be the same as `keycloak.protect()`. it will check if a user was authenticated and I can also add a check to check for roles as well. Will change the name to `protect` instead. I'm not sure what the `login-required` for keycloak does?